### PR TITLE
Paid content in rich links

### PR DESCRIPTION
**Pull request by lps88**
## What does this change?
Add styling for paid-for articles in rich links.

## Screenshots
image

as compared to editorial content:
image

## Tested in CODE?
No

### CODE REVIEW

**Comment by shtukas** [comment]
Looks good to me :+1:, but having @rich-nguyen second opinion would be nice :)

**Review by rich-nguyen — APPROVED** [review]
sweet

File: static/src/stylesheets/module/onward/_rich-links-enhanced.scss [hunk 1]
```diff
@@ -200,3 +200,30 @@
     float: none;
     width: auto;
 }
+
+.rich-link--paidfor {
+    &.tone-media--item,
```
Comment (rich-nguyen) [inline static/src/stylesheets/module/onward/_rich-links-enhanced.scss]: why is this selector needed? overrides?

Comment (lps88) [inline static/src/stylesheets/module/onward/_rich-links-enhanced.scss]: Yeah, the media tone has high(ish) specificity selectors. Not sure at the moment if we want different styles for paid galleries/paid articles so they're both the same for now.

Open to suggestions for a neater way to do this though! :)

**Pull request merged**